### ISSUE TITLE
readme mention style.css but doesn't exist

### ISSUE THREAD

**Issue by jacobanana** [youngpada1/portfolio-app#12]
The readme file mentions a `style.css` file to edit the styling to my liking, but this file doesn't exist in the repository.

> 3. Customize the `style.css` file to match your preferred color scheme and styling. 

Should this be referring us to the `.streamlit/config.toml` file or is there a way to add a CSS file for styling?

Streamlit [theming](https://docs.streamlit.io/library/advanced-features/theming) options are pretty limited :(

**Comment by youngpada1** [comment]
Yup, you're right! There's a way to style it but it's annoying. I'll change that..also yeah, theming sux!
